scripts/dm-swarm.sh

```diff
@@ -1,23 +1,75 @@
 #!/usr/bin/env bash
+# Exit When Any Command Fails
```

Comment:
This script is amazing. Great work!

However, it's too risky to change it that much given that it is mentioned in many places in docs, books, and videos. People will get confused with such an amount of changes when compared with the explanations in those docs, books, and videos. 

How about creating a new file instead (e.g., `dm-swarm-02.sh`)? That way it wouldn't matter where one, some, or all the places where the old script is mentioned are updated to the new one.